codeJK2/game/CMakeLists.txt

```diff
@@ -232,7 +232,11 @@ set(JK2SPGameCommonFiles
 	"${SPDir}/qcommon/q_shared.cpp"
 	"${SPDir}/qcommon/q_shared.h"
 	"${SPDir}/qcommon/strippublic.h"
-	
+	"${SPDir}/rd-common/mdx_format.h"
```

Comment:
The game module shouldn't be including renderer headers apart from tr_public.h. Why is this being included?